netlify.toml

```diff
@@ -0,0 +1,7 @@
+[build]
+    base = "react/"
+    publish = "storybook-static"
+    command = "npm run build-storybook"
+[context.deploy-preview]
+    # Skip preview build if there are no changes to mdx files in react/src
+    ignore = "curl https://api.github.com/repos/jasonchong96/ogp-design-system-test/pulls/$REVIEW_ID/files | grep -L '\"filename\": \"react/src/.*.mdx\"'"
```

Comment:
should we make this more generic? can we use github env vars instead?

Reply:
Looked at the env vars we can automatically access on Netlify:
https://docs.netlify.com/configure-builds/environment-variables/#netlify-configuration-variables

I've pushed a commit to get `REPOSITORY_URL` and use it to "derive" the api.github.com url to fetch the commit instead. This way, there is no need to change anything when the repository changes.